ModelicaTest/Media.mo

```diff
@@ -950,6 +950,22 @@ is given to compare the approximation.
       annotation (experiment(StopTime=1));
     end R134a_setState_pTX;
 
+    model R134a_setState_phX "Test setState_phX() of R134a"
+      extends Modelica.Icons.Example;
+      replaceable package Medium = Modelica.Media.R134a.R134a_ph "Medium model";
+      SI.Temperature h = Medium.h_default;
+      parameter SI.AbsolutePressure p0 = 10e5 "p at time 0";
+      parameter Real pRate(unit = "Pa/s") = 20e5 "p's rate of change";
```

Comment:
```suggestion
      parameter SI.PressureSlope pRate = 20e5 "p's rate of change";
```

Reply:
It was copied from R134a_setState_pTX which hasn't been updated either. I think we should coordinate that later.

Reply:
Here you go: #4273